### PR TITLE
add full links to freek.yaml

### PR DESCRIPTION
**Pull request by khanh93**
cf. this conversation
https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/freek.2Eyaml/near/197670168

### CODE REVIEW

File: data/100.yaml [hunk 1]
```diff
@@ -170,11 +170,11 @@
       - author :
    51:
       - title  : Wilson’s Theorem
-      - decl   : wilsons_lemma
+      - decl   : number_theory/quadratic_reciprocity.html#zmod.wilsons_lemma
       - author : Chris Hughes
    52:
       - title  : The Number of Subsets of a Set
-      - decl   : card_powerset
+      - decl   : data/multiset.html#multiset.card_powerset
```
Comment (urkud) [inline data/100.yaml:177]: I'd list `finset` version here.

File: data/100.yaml [hunk 2]
```diff
@@ -220,29 +220,29 @@
...
       - author :
    71:
       - title  : Order of a Subgroup
-      - decl   : card_subgroup_dvd_card
+      - decl   : group_theory/order_of_element.html#card_subgroup_dvd_card
       - author : mathlib
    72:
       - title  : Sylow’s Theorem
-      - decl   : [exists_subgroup_card_pow_prime, sylow_conjugate, card_sylow_dvd, card_sylow_modeq_one]
+      - decl   : [group_theory/sylow.html#sylow.exists_subgroup_card_pow_prime, sylow_conjugate, card_sylow_dvd, card_sylow_modeq_one]
```
Comment (jcommelin) [inline data/100.yaml:245]:
```suggestion
      - decl   : [group_theory/sylow.html#sylow.exists_subgroup_card_pow_prime, group_theory/sylow.html#sylow_conjugate, group_theory/sylow.html#card_sylow_dvd, group_theory/sylow.html#card_sylow_modeq_one]
```

File: data/100.yaml (hunk 1 above)
Comment (khanh93) [reply data/100.yaml:177]: Seems correct. Once I figure out how to update it, it should be
data/finset.html#finset.card_powerset

Comment (khanh93) [reply data/100.yaml:177]: Does this do it
```suggestion
      - decl   : data/finset.html#finset.card_powerset
```

**Pull request merged**